CONTRIBUTING.rst

```diff
@@ -1,461 +1,465 @@
-============
-Contributing
-============
-
-Contributions are welcome, and they are greatly appreciated! Every
-little bit helps, and credit will always be given. 
-
-.. toctree::
-   :numbered:
-   :maxdepth: 2
-
-   types_of_contributions
-   contributor_setup
-   contributor_guidelines
-   contributor_testing
-   core_committer_guide
-
-
-Types of Contributions
-----------------------
-
-You can contribute in many ways:
-
-Create Cookiecutter Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Some other Cookiecutter templates to list in the :ref:`README <readme>` would
-be great.
-
-If you create a Cookiecutter template, submit a pull request adding it to
-README.rst.
-
-Report Bugs
-~~~~~~~~~~~
-
-Report bugs at https://github.com/audreyr/cookiecutter/issues.
-
-If you are reporting a bug, please include:
-
-* Your operating system name and version.
-* Any details about your local setup that might be helpful in troubleshooting.
-* If you can, provide detailed steps to reproduce the bug.
-* If you don't have steps to reproduce the bug, just note your observations in
-  as much detail as you can. Questions to start a discussion about the issue
-  are welcome.
-
-Fix Bugs
-~~~~~~~~
-
-Look through the GitHub issues for bugs. Anything tagged with "bug"
-is open to whoever wants to implement it.
-
-Implement Features
-~~~~~~~~~~~~~~~~~~
-
-Look through the GitHub issues for features. Anything tagged with "enhancement"
-and "please-help" is open to whoever wants to implement it.
-
-Please do not combine multiple feature enhancements into a single pull request.
-
-Note: this project is very conservative, so new features that aren't tagged
-with "please-help" might not get into core. We're trying to keep the code base
-small, extensible, and streamlined. Whenever possible, it's best to try and 
-implement feature ideas as separate projects outside of the core codebase.

```

Comment:
@michaeljoseph OK, I double-checked, and I found that both the commit history and the raw content of the file `docs/types_of_contributions.rst` on both `audreyr/master` and `stevepiercy/rest-syntax` are identical.

Compare history:
* https://github.com/audreyr/cookiecutter/commits/master/docs/types_of_contributions.rst
* https://github.com/stevepiercy/cookiecutter/commits/rest-syntax/docs/types_of_contributions.rst

Compare source:
* https://raw.githubusercontent.com/audreyr/cookiecutter/master/docs/types_of_contributions.rst
* https://raw.githubusercontent.com/stevepiercy/cookiecutter/rest-syntax/docs/types_of_contributions.rst

Do you see something that I missed?

In the commit you linked to, it appears that @audreyr on Jun 7, 2016 made the commit to only 1 file, `/CONTRIBUTING.rst`, but not the source file.

Would you like me to take the changes from @audreyr's commit in `/CONTRIBUTING.rst` and put them into `docs/types_of_contributions.rst` on my branch, then render `/CONTRIBUTING.rst` again and commit both files? I think that is what you want in the end, but want to confirm. Thank you!